ops/jenkins/global-pipeline-libraries/vars/awsSsm.groovy

```diff
@@ -18,11 +18,16 @@ String putParameter(Map args = [:]) {
     name = args.parameterName
     value = args.parameterValue
     type = args.parameterType ?: 'String'
+    tags = args.parameterTags ?: "Key=Source,Value=${JOB_NAME} Key=Environment,Value=mgmt Key=stack,Value=mgmt Key=Terraform,Value=False Key=application,Value=bfd Key=business,Value=oeda"
     overwrite = args.shouldOverwrite ? '--overwrite' : ''
     awsRegion = args.awsRegion ?: 'us-east-1'
     includeType = "--type ${type}"
```

Comment:
I would either move `--type` into the cli call, or include the opt prefix in all of these variables to be consistent. e.g. 

```
typeOpt = args.parameterType ?: '--type String'
tagsOpt = args.parameterTags ?: "--tags Key=Source,Value=${JOB_NAME} Key=Environment,Value=mgmt Key=stack,Value=mgmt Key=Terraform,Value=False Key=application,Value=bfd Key=business,Value=oeda"
overwriteOpt = args.shouldOverwrite ? '--overwrite' : ''
awsRegionOpt = args.awsRegion ?: '--region us-east-1'
typeOpt = "--type ${type}"

rTypeOpt = args.resourceType ?: '--resource-type Parameter'
``